src/CpmsClient/Client/RestClientFactory.php

```diff
@@ -25,10 +27,15 @@ class RestClientFactory implements FactoryInterface
      * @return HttpRestJsonClient
      * @throws \Psr\Container\ContainerExceptionInterface
      * @throws \Psr\Container\NotFoundExceptionInterface
+     *
+     * Required suppression due to un-typed parameter in parent class
+     * @psalm-suppress MissingParamType
```

Comment:
Would it worked if we type @param $requestedName as mixed?

Reply:
it does :)